packets/packets.go

```diff
@@ -5,12 +5,12 @@ import (
 	"io"
 )
 
-const ProtocolVersion = 753
+const ProtocolVersion = 755
 const MaxPacketSize = 128 * 1024
 const MaxPacketID = 256
-const CompressThreshold = 512
+const CompressThreshold = 256
```

Comment:
1.17 nie wspiera sensownego compress threshold ze zmniejszone?

Reply:
Domyślna wartość network-compression-threshold w server.properties to 256

Reply:
Mogę przywrócić na 512 jeżeli to problem

Reply:
Tak, nie jest to związane z tematem PR. Nie jest to też zmiana na plus. Nie potrzebujemy kompresować takich mini pakietów.